compose.yml

```diff
@@ -0,0 +1,118 @@
+services:
+  web:
+    build:
+      context: "web"
+      target: "production"
+    ports:
+      - "3000:3000"
+    healthcheck:
+      test: ["CMD-SHELL", "wget --spider http://localhost:3000 || exit 1"]
+    env_file: [".env"]
+    depends_on:
+      db-psql:
+        condition: "service_healthy"
+      temporal:
+        condition: "service_healthy"
+  db-psql:
+    image: "postgres:18.1-alpine"
+    environment:
+      POSTGRES_USER: "${DB_USER:-}"
+      POSTGRES_PASSWORD: "${DB_PASSWORD:-}"
+      POSTGRES_DB: "${DB_NAME:-}"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${DB_USER:-} -d ${DB_NAME:-}"]
+    volumes:
+      - "db-psql:/var/lib/postgresql/18/docker"
```

Comment:
This mount needs to be targeting `/var/lib/postgresql` instead, same for temporal-db.

Targeting `/var/lib/posgtgresql` will likely help when we need to migrate to 19 in the future